app/src/main/java/com/nervousfish/nervousfish/activities/ContactActivity.java

```diff
@@ -0,0 +1,61 @@
+package com.nervousfish.nervousfish.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
+
+import com.nervousfish.nervousfish.ConstantKeywords;
+import com.nervousfish.nervousfish.R;
+import com.nervousfish.nervousfish.service_locator.IServiceLocator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The ContactActivity shows the contacts information and his public keys.
+ */
+@SuppressWarnings("PMD.AtLeastOneConstructor")
+public final class ContactActivity extends AppCompatActivity {
+
+    private final List<String> keys = new ArrayList<>();
+    @SuppressWarnings("PMD.SingularField")
```

Comment:
Could you add for this suppression and the suppression above a very short message describing why you suppressed the warning?